src/api/app/models/concerns/staging_project.rb

```diff
@@ -87,19 +89,30 @@ def broken_packages
     @broken_packages
   end
 
-  def missing_reviews
-    return @missing_reviews if @missing_reviews
+  def missing_reviews_for_classified_requests(request, reviews)
 
-    @missing_reviews = []
+    @missing_reviews_of_st_project ||= []
 
-    Review.includes(:bs_request).where(bs_request_id: staged_requests.select(:id)).where.not(state: :accepted).find_each do |review|
-      # We skip reviews for the staging project since these reviews are used
-      # by the openSUSE release tools _after_ the overall_state switched to
-      # 'accepted'.
-      next if review.by_project == name
-      extract_missing_reviews(review.bs_request, review)
+    current_missing_reviews = reviews.reduce([]) do |collected, review|
```

Comment:
If this reduce is the last expression in the function, you can remove the `current_missing_reviews` assignment, you know, the reduce's return value will be the surrounding function's implicit return value.

Reply:
yeah, rubocop allready complained. Rubocop also doesnt like the reduce, so I switch to ```each_with_object```

Reply:
Okay